package.json

```diff
@@ -5,7 +5,12 @@
     "rebuild:browser": "theia rebuild:browser",
     "rebuild:electron": "theia rebuild:electron",
     "start:browser": "yarn rebuild:browser ; yarn --cwd browser-app start",
-    "start:electron": "yarn rebuild:electron ; yarn --cwd electron-app start"
+    "start:electron": "yarn rebuild:electron ; yarn --cwd electron-app start",
+    "download:sample-traces": "git clone https://github.com/tracecompass/tracecompass-test-traces.git; ln -s tracecompass-test-traces/ctf/src/main/resources sample-traces",
+    "download:server": "wget http://www.eclipse.org/downloads/download.php?file=/tracecompass.incubator/trace-server/rcp/trace-compass-server0.0.1-20200603-0829-linux.gtk.x86_64.tar.gz",
```

Comment:
Unfortunately this does not work - the link needs to be accessed with a browser apparently. Where else can we find the trace server?

Reply:
You can just prepend the filename after the file= with https://download.eclipse.org/, like this https://download.eclipse.org/tracecompass.incubator/trace-server/rcp/trace-compass-server0.0.1-20200603-0829-linux.gtk.x86_64.tar.gz

The problem is that each day [where there is a new build] the name of the build changes, I don't know if we can create a symlink, trace-server-latest with a redirect that will always point to latest trace server, or else we'll have to do releases

Reply:
Thanks @tahini - I have something that works locally - I'll test on Gitpod later.

Reply:
> Thanks for this. Does your second commit means we'll have a demo of the trace viewer in gitpod? That's awesome!

Yes, that's the idea. We're one step closer I think, with the proposed URL to download the server now working.

Reply:
> You can just prepend the filename after the file= with https://download.eclipse.org/, like this https://download.eclipse.org/tracecompass.incubator/trace-server/rcp/trace-compass-server0.0.1-20200603-0829-linux.gtk.x86_64.tar.gz
> 
> The problem is that each day [where there is a new build] the name of the build changes, I don't know if we can create a symlink, trace-server-latest with a redirect that will always point to latest trace server, or else we'll have to do releases

Ah, I think I see now. I took care of our side of things by renaming the downloaded server archive, but I guess the file name changes on the server side too. Maybe your idea of providing an unchanging download name, with `latest` instead of a specific version, would be required? 

@bhufmann is that something you could look into?

Reply:
I have a patch that does that here: https://git.eclipse.org/r/#/c/164948/

Now we have to see if it will work with the symlink or if we'd need to copy the whole file

Reply:
Cool - can you let me know when your patch is merged and I'll use the new name.